src/styles/blocks.scss/cards.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: #{$cards-size-width};
+  padding: 0 16px;
+  padding-bottom: 46px;
+  margin-right: $cards-margin-rights;
```

Comment:
Still, the BEM block can't have a margin.
Maybe, create page__card selector and put these style there ?

Reply:
i put selector to section and give to this selector margin

> class='card card__page card--3'